src/Whmcs/Provider.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace SocialiteProviders\Whmcs;
+
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+/**
+ * https://docs.whmcs.com/OpenID_Connect_Developer_Guide
```

Comment:
```suggestion
 * @see https://docs.whmcs.com/OpenID_Connect_Developer_Guide
```
Awesome idea to add this!